core/modules/user.py

```diff
@@ -297,8 +288,12 @@ def pwrecover(self, *args, **kwargs):
                         tpl=self.passwordRecoveryFailedTemplate,
                         reason=self.passwordRecoveryKeyInvalid)
                 return self.userModule.render.edit(skel, self.passwordRecoveryStep2Template)  # Let's try again
+
             # If we made it here, the key was correct, so we'd hopefully have a valid user for this
-            uSkel = userSkel().all().filter("name.idx =", session["user.auth_userpassword.pwrecover"]["name"]).getSkel()
+            uSkel = self.viewSkel().all().filter(
+                "name.idx =", session["user.auth_userpassword.pwrecover"]["name"]
+            ).getSkel()
+
             if not uSkel:  # This *should* never happen - if we don't have a matching account we'll not send the key.
```

Comment:
Maybe we can rename the variable here and lower in the code like :
https://peps.python.org/pep-0008/#function-and-variable-names
```suggestion
            user_skel = self.viewSkel().all().filter(
                "name.idx =", session["user.auth_userpassword.pwrecover"]["name"]
            ).getSkel()

            if not user_skel:  # This *should* never happen - if we don't have a matching account we'll not send the key.
```

Reply:
Maybe we should discuss if we rename the most vars in this they all look like:
```python
    registrationEnabled = False
    registrationEmailVerificationRequired = True
    registrationAdminVerificationRequired = True
    verifySuccessTemplate = "user_verify_success"
    verifyEmailAddressMail = "user_verify_address"
    verifyFailedTemplate = "user_verify_failed"
    passwordRecoveryTemplate = "user_passwordrecover"
    passwordRecoveryMail = "user_password_recovery"
    passwordRecoveryAlreadySendTemplate = "user_passwordrecover_already_sent".......
```

But I dont't now if this are breaking changes.

Reply:
Hello Arne,

I'm totally with you. I think these special templates aren't used that much in current VIUR3 systems.

Anyway, if we provide a generalized deprecation handling for module members, we can do it without totally breaking the system. Such a generalized handling can be placed in the BasicApplication prototype, and customized via a replacement dict. This replacement dict is filled when renaming module-specific members (including functions!).

I'll make an issue on this and start a PR which we can use for further discussion.

Reply:
I've opened issue #594 on this. This change including a renaming is bigger and also involves rendering code, therefore I wouldn't integrate it within this pull request. I've started this pull request as preliminary work for #587.